src/parsers.ts

```diff
@@ -0,0 +1,110 @@
+import { xdr, SorobanDataBuilder } from 'stellar-base';
+import { SorobanRpc } from './soroban_rpc';
+
+export function parseLedgerEntries(
```

Comment:
since this method is exported, shoiuld it get jsdoc'd?

Reply:
ugh, I only export it for testing purposes, so it's not valuable to document (plus I'm too lazy)

Reply:
oh wait, it's not exported on a library level, only in the module file, so no need to jsdoc anyway